parm/config/config.fcst

```diff
@@ -138,6 +138,31 @@ export ishuffle=0
 export shuffle=1
 export deflate_level=1
 
+export OUTPUT_FILETYPES="$OUTPUT_FILE"
+if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
+    export  ichunk2d=0; export jchunk2d=0
+    export  ichunk3d=0; export jchunk3d=0;  export kchunk3d=0
+    RESTILE=`echo $CASE |cut -c 2-`
+    if [[ "$machine" == "WCOSS_DELL_P3" ]]; then
+        if [ $RESTILE -ge 768 ]; then
+            export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf_parallel' "
+            export ichunk3d=$((4*RESTILE)) 
+            export jchunk3d=$((2*RESTILE))
+            export kchunk3d=1
+        else
+            export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf' "
+        fi
+    fi
+    if [[ "$machine" == "HERA" ]]; then
```

Comment:
Why is Hera different here? Mostly just curious.

Reply:
This setting is based on extensive tests made by Jun Wang and Jeff Whitaker.  It is odd, but indeed writing the 2D files using parallel_netcdf is slow than serial netcdf on WCCOSS but faster on HERA.

Reply:
Got it, thanks!